po/bn.po

```diff
@@ -251,431 +251,431 @@ msgstr "দিন 2: সকাল"
 
 #: src/SUMMARY.md:76
 msgid "Structs"
-msgstr "কাঠামো"
+msgstr "Structs"
 
 #: src/SUMMARY.md:77
 msgid "Tuple Structs"
 msgstr "Tuple কাঠামো"
 
 #: src/SUMMARY.md:78
 msgid "Field Shorthand Syntax"
-msgstr ""
+msgstr "Field Shorthand Syntax"
 
 #: src/SUMMARY.md:79
 msgid "Enums"
-msgstr ""
+msgstr "Enums"
 
 #: src/SUMMARY.md:80
 msgid "Variant Payloads"
-msgstr ""
+msgstr "ভিন্ন Payloads"
 
 #: src/SUMMARY.md:81
 msgid "Enum Sizes"
-msgstr ""
+msgstr "Enum এর মাপ"
 
 #: src/SUMMARY.md:83
 msgid "Method Receiver"
-msgstr ""
+msgstr "Receiver পদ্ধতি"
```

Comment:
"method" can't be directly translated to পদ্ধতি, as I don't think this is a well established translation of the programming term.